exercises/04_power/power_test.go

```diff
@@ -2,14 +2,25 @@ package main
 
 import "testing"
 
+//
 // TestPow tests a Pow(x, y) function, which returns integer x**y.
 func TestPow(t *testing.T) {
 	for _, tc := range []struct {
 		x, y uint
 		want uint
 	}{
-		// Test cases.
+		{x: 1, y: 2, want: 1},
```

Comment:
Non-tested scenarios:

* `y = 0`
* `x` is negative
* `y` is negative.

Reply:
Ah, please ignore `y` is negative.

Reply:
And you also miss a very obvious case - zero values for both `x` and `y`